README.MD

```diff
@@ -54,6 +54,7 @@ MyUni demonstrates the following:
 4. Install React dependencies using [npm](https://www.npmjs.com/) package manager  **npm install**
 5. Update the **.env** file with the integration key and other settings.  
      > **Note:** Protect your integration key and client secret. You should make sure that your **.env** file will not be stored in your source code repository.
+6. Navigate to [the admin demo Apps and Keys page](https://admindemo.docusign.com/apps-and-keys), add the Redirect URI http://localhost:3000/callback and then hit save
```

Comment:
https://admindemo.docusign.com/authenticate?goTo=appsAndKeys is a better link (the one you had may require you to log in and not take you to where you want)